aptos-move/move-examples/post_mint_reveal_nft/sources/whitelist.move

```diff
@@ -0,0 +1,102 @@
+module post_mint_reveal_nft::whitelist {
+    use post_mint_reveal_nft::bucket_table::BucketTable;
+    use std::vector;
+    use std::error;
+    use post_mint_reveal_nft::bucket_table;
+    use std::signer;
+    use aptos_framework::timestamp;
+    use aptos_framework::account;
+    friend post_mint_reveal_nft::minting;
+
+    /// WhitelistMintConfig stores information about all stages of whitelist.
+    /// Most whitelists are one-stage, but we allow multiple stages to be added in case there are multiple rounds of whitelists.
+    struct WhitelistMintConfig has key {
+        whitelist_configs: vector<WhitelistStage>,
+    }
+
+    /// WhitelistMintConfigSingleStage stores information about one stage of whitelist.
+    struct WhitelistStage has store {
+        whitelisted_address: BucketTable<address, u64>,
+        whitelist_mint_price: u64,
+        whitelist_minting_start_time: u64,
+        whitelist_minting_end_time: u64,
+    }
+
+    const EINVALID_WHITELIST_SETTING: u64 = 1;
```

Comment:
Can you add a comment explaining each of these error? These comments are used to generate the errors when txs failed with these error codes.

Reply:
yup will update